qt/aqt/data/web/css/reviewer-bottom.scss

```diff
@@ -28,6 +28,34 @@ button {
     white-space: nowrap;
     margin: 9px;
     position: relative;
+    border-top-color: color(border-subtle);
```

Comment:
```suggestion
```

I think this shouldn't be here and is from Expertium misusing a diff (it's copied from the `.nightMode` block at the bottom)